src/main/java/org/mule/extension/aggregator/internal/privileged/executor/AbstractAggregatorExecutor.java

```diff
@@ -201,8 +206,12 @@ public void start() throws MuleException {
     if (clusterService.isPrimaryPollingInstance()) {
       if (!started) {
         startIfNeeded(objectStore);
-        upgradeAggregatedContentIfNeeded();
-        setRegisteredAsyncAggregationsAsNotScheduled();
+        if (failOnStartIfNoQuorum) {
+          deferredActions();
```

Comment:
What's happened with the exceptions here? Shouldn't we log something? Even, if we have a quorum exception we can suggest using the new system property to solve the problem.

Reply:
we should not change the current logic at this point in the module